maven-artifact/src/main/java/org/apache/maven/artifact/handler/ArtifactHandler.java

```diff
@@ -19,10 +19,10 @@
 package org.apache.maven.artifact.handler;
 
 /**
- * An artifact handler defines for a dependency type, defined as Plexus role:<ul>
- * <li>extension and classifier, to be able to download the file,</li>
+ * An artifact handler contains metadata derived from the dependency element that references the artifact:<ul>
```

Comment:
I don't think that's true.
The metadata is not `derived from the dependency element`.  The set of artifact handlers is reduced and provides metadata related to the `dependency type`.